hamilflow/models/particle_in_box.py

```diff
@@ -0,0 +1,143 @@
+from functools import cached_property
+from typing import Dict, List, Optional, Union
+
+import numpy as np
+import pandas as pd
+import scipy as sp
+from pydantic import BaseModel, Field, computed_field, field_validator
+
+
+PLANCK_CONST = 6.626e-34
```

Comment:
https://docs.scipy.org/doc/scipy/reference/constants.html#physical-constants

Alternatively we may let the use decide the unit. Tiny numbers can become impractical.

Reply:
Yeah, too small numbers could be impractical. Another option is not to use planck constant explicitly. Or do planck related calculations at the very end.